contribution/plan.md

```diff
@@ -0,0 +1,11 @@
+Project Contributions
+
+[First Contribution](https://github.com/danthareja/contribute-to-open-source/issues/1)
```

Comment:
pick at least one project and setup a markdown file for it, see other PRs as example

Reply:
I picked one of the javascript projects for the contribution project and attached the link as the professor's md file.